src/metaxy/graph/diff/models.py

```diff
@@ -13,6 +13,11 @@
     from metaxy.models.feature import FeatureGraph
 
 
+# Sentinel value for None code_version in struct serialization
+# (Polars structs require a non-None value, so we use "0" to represent missing/None)
+EMPTY_CODE_VERSION = "0"
```

Comment:
and this is something we should move over to a shared utils so it is not re-defined